src/modules/0.0.12/aws_ec2/index.ts

```diff
@@ -15,7 +15,6 @@ import * as metadata from './module.json'
 import { AwsElbModule } from '../aws_elb'
```

Comment:
I think the equals method fro instance should be updated here

Reply:
Thank you for the hint :)